src/tarkov-data-manager/jobs/update-quests.js

```diff
@@ -1374,17 +1398,18 @@ const zoneMap = {
     quest_zone_keeper7_saferoom: '5b0fc42d86f7744a585f9105', //labs
     quest_zone_keeper7_test: '5b0fc42d86f7744a585f9105',
     tadeush_bmp2_area_mark_12: '5704e5fad2720bc05b8b4567',
+    ter_017_area_1: '59fc81d786f774390775787e',
 };
 
 const questItemLocations = {
     '5968929e86f7740d121082d3': '56f40101d2720b2a4d8b45d6', // customs
     '6398a4cfb5992f573c6562b3': '5b0fc42d86f7744a585f9105', //labs
     '6398a0861c712b1e1d4dadf1': '5704e4dad2720bb55b8b4567', //lighthouse
     '6398a072e301557ae24cec92': '5704e5fad2720bc05b8b4567', // reserve
-    '5af04c0b86f774138708f78e': '56f40101d2720b2a4d8b45d6', //woods
+    '5af04c0b86f774138708f78e': '5704e3c2d2720bac5b8b4567', //woods
     '5b4c72b386f7745b453af9c0': '5704e554d2720bac5b8b456e', // shoreline
     '5b4c72c686f77462ac37e907': '5704e554d2720bac5b8b456e',
```

Comment:
It might be nice to add comments like we do above for each of these lines so we can quickly tell what map these are